src/components/Header.js

```diff
@@ -1,10 +1,50 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({sender, senderImg, changeSender}) {
+	
```

Comment:
저는 arrow function을 자주 사용해서 저는 편해서 거의 주로 그렇게 쓰거든요!
```suggestion
const Header = ({sender, senderImg, changeSender}) =>{   내용  } 
export default Header;
```
이런식으로 했고 , 물론 다른 function 모두 이런식으로( => 를 통해) 표현이 가능합니다!! 
알고 계셨다면 TMI네요 😄

Reply:
좋은 조언 감사합니다! 뭔가 전 arrow function 표현이 아직 낯설어서 잘 활용을 못하는 것 같아요.. (쪼랩)